### ISSUE TITLE
Immortal and silent mimics

### ISSUE THREAD

**Issue by Sora-Akuma** [TimShaw1/Wendigos-Mod#14]
The first mimic will be normal, but all subsequent ones will be invulnerable and will not reproduce the voice.

Version Game: Lethal Company v62; Mod launcher: r2modmain
The only mods included were 
- BepInExPack 
- Wendigos Voice Cloning

**Comment by MetaKoopa** [comment]
It looks like I'm experiencing this as well. I was reviewing the LogOutput.log from r2modman and found this which I think might be relevant & occurs anytime a player is infected by a Masked/Mimic. The IMPERIUM line is related to Imperium debugger which I was using for troubleshooting. 

```
[Warning: Unity Log] HeadMaskTragedy is disabled! Netcode for GameObjects does not support spawning disabled NetworkBehaviours! The RandomPeriodicAudioPlayer component was skipped during spawn!
[Info   :  Imperium]  - SPENT IN VISUALIZATION: 0
[Info   :   Console] Wendigos: Spawned Masked. ID: (16.50, -0.21, -24.55)
[Info   :   Console] Wendigos: System.Collections.Generic.List`1[System.UInt64]
[Info   :   Console] Wendigos: Created unasssigned list
```
I've found that it seems to work properly for mimics which are not spawned from a player.

**Comment by TimShaw1** [comment]
> The first mimic will be normal, but all subsequent ones will be invulnerable and will not reproduce the voice.
> 
> Version Game: Lethal Company v62; Mod launcher: r2modmain The only mods included were
> 
> * BepInExPack
> * Wendigos Voice Cloning

Subsequent mimics now cloning voices is by design. I have never had mimics become invincible in my testing, what mod are you using to spawn them?

**Comment by TimShaw1** [comment]
Fixed in 1.0.9
